modules/ROOT/pages/setup.adoc

```diff
@@ -95,41 +95,32 @@ Anypoint Partner Manager enables you to connect your own storage solution (such
 You'll need to establish an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage.
 The follow the steps explain how to setup the Payload Storage API:
 
-. Download the Partner Manager Content Storage API and sample implementation template from Anypoint Exchange.
-Install this API on a Mule runtime that can access the storage solution.**[MISSING INFO - Confirm final name by GA. Add link if it's live before GA.]**
+. Download the *Partner Manager Content Storage API* and sample implementation template from Anypoint Exchange.
+Install this API on a Mule runtime that can access the storage solution.
 
-. Make a note of the API address configurations (make a note of the following):
+. Make a note of the API address configurations, for example:
 
-** host, port, path connection
+* host, port, path connection
 
-** response timeouts
+* response timeouts
 
-** authentication details: None, API Key, Basic, Client Credentials, OAuth Token
+* authentication types: None, API Key, Basic, Client Credentials, OAuth Token
 
-** Certificate (.PEM format) and connection type
+* Certificate (`.PEM` format) and connection type
 
-[xref] to HTTPS Send Configuration for complete details?
-
-You'll need these details while associating Anypoint Partner Manager to your Payload Storage API. +
+You'll need this information while associating Anypoint Partner Manager to your Payload Storage API. +
 
 NOTE: Your payload storage API must be accessible over the internet via HTTPS.
 Ensure that the appropriate authentication mechanisms have been implemented to make this happen.
 
-. Once the Payload Storage API is set up on the Mule runtime, open Anypoint Partner Manager, navigate to your organizations profile (Host) page from the left navigation.
+. Once the Payload Storage API is set up on the Mule runtime, open Anypoint Partner Manager, locate to your organizations profile (Host) page from the left navigation.
 
  . Select the `Payload storage` section and configure the path to your storage.
 
 You are ready to begin to set up message flows between you and your trading partners.
```

Comment:
Is this true? I can't tell where in the User Journey this file is, but the user has to do these things in this order???

Configure Host
Configure Partner
Configure Flow
Configure Anypoint Platform for PM

Is this the flow? Or does the order not matter? I can't remember if I have seen this specified anywhere, but it should be, right from the beginning.

Reply:
The user must configure the platform before doing the first three.